test/Microsoft.Framework.Configuration.Binder.Test/ConfigurationCollectionBindingTests.cs

```diff
@@ -182,9 +183,9 @@ public void StringDictionaryBinding()
 
             Assert.Equal(3, options.StringDictionary.Count);
 
-            Assert.Equal("val_1", options.StringDictionary["abc"]);
-            Assert.Equal("val_2", options.StringDictionary["def"]);
-            Assert.Equal("val_3", options.StringDictionary["ghi"]);
+            Assert.Equal("val_1", options.StringDictionary["StringDictionary:abc"]);
```

Comment:
Why did the keys change here?  That seems like a regression in behavior for all of the keys to have changed... was that intentional?